### ISSUE TITLE
Examples

### ISSUE THREAD

**Issue by shiffman** [ITPNYU/ICM-2015#11]
Using p5.js we don't have textbooks and piles of examples.  There are many at these sources:

http://p5js.org/examples/
https://github.com/shiffman/LearningProcessing-p5.js
https://github.com/shiffman/The-Nature-of-Code-Examples-p5.js

As we work week to week, we should create a directory for each week and place additional examples (made in advance and in class).  

What am I missing for sources for examples?

**Comment by lmccart** [comment]
https://github.com/lmccart/itp-networked-media
https://github.com/lmccart/itp-creative-js
https://github.com/lmccart/itp-critical-apis
https://github.com/lmccart/gswp5.js-code

**Comment by shiffman** [comment]
Ok, @lmccart just discussed this.  What I did for now is put all examples into a folder called https://github.com/ITPNYU/ICM-2015/tree/master/01_drawing.  Then I zipped that folder and linked the syllabus week's header to that folder.   See:

https://github.com/ITPNYU/ICM-2015/blob/master/README.md#1----introduction-and-drawing

(Buried below is also a "link to examples" link).

As we add and change examples, we'll likely need to update the zip file.  Sound good?
